l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -261,7 +261,24 @@ def to_xml(self, env):
             # i controlli precedenti dovrebbero escludere errori di sintassi XML
             # with open("/tmp/fatturaout.xml", "wb") as o:
             #    o.write(etree.tostring(root, xml_declaration=True, encoding="utf-8"))
-            raise UserError("\n".join(str(e) for e in errors))
+            # Print error paths, as they can be helpful even for non-technical people
+            error_path_string = "\n- ".join(
+                err.path[err.path.index(":") + 1 :] for err in errors
```

Comment:
Anche `err.reason` potrebbe essere utile, ma probabilmente genererebbe solo confusione in un utente non tecnico perché contiene cose tipo
> value doesn't match any pattern of ['[\\p{IsBasicLatin}\\p{IsLatin-1Supplement}]{1,60}']

quindi ok così.